Create ui-c8y-1022-47-0-reordering-properties-in-asset-properties-selector.md (#3888)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-47-0-reordering-properties-in-asset-properties-selector.md

---
date: ""

> title: Asset property selector allows reordering properties using drag and drop

Reviewer: ```suggestion
title: Asset property selector allows reordering of properties
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64675
version: 1022.47.0
---

> The asset property selector is used to select properties to be displayed in various places in the application. Previously, the order of the properties could not be changed. With this update, it is now possible to reorder the properties in the asset property selector using drag and drop. This allows users to customize the order in which properties are displayed to better suit their needs and preferences.

Reviewer: ```suggestion
The asset property selector is used to select properties to be displayed in various places in the platform. Previously, the order of the properties could not be changed. With this update, you can now reorder the properties in the asset property selector using drag and drop. This allows users to customize the order in which properties are displayed to better suit their needs and preferences.
```
